### PR TITLE
Update RNA-seq Docker image and descriptor schema

### PR DESCRIPTION
**Pull request by mzganec**

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -24,6 +24,8 @@ Changed
 * Set core resource requirement in ``cuffnorm`` process to 1
 * Add sample descriptor to ``prepare_expression`` test function
 * Add access to sample field of data objects in processes
+* Add ``HISAT2`` and ``RSEM`` to ``resolwebio/rnaseq`` Docker image
```
Comment (jkokosar) [inline docs/CHANGELOG.rst]: Reference to this image is now under added and changed section, all within a single release. I suggest you omit this, or modify the original change-log entry .

**Comment by mzganec** [comment]
@jkokosar I removed the added item about RNA-seq docker image from the change log.

**Comment by mstajdohar** [comment]
Please rebase.

**Comment by jkokosar** [comment]
@mzganec @tjanez There is an issue with using a system package which would downgrade the installed STAR aligner tool. In the new releases of STAR some issues were fixed that could affect the analysis we are targeting, e.g.:

Fixed a bug in --quantMode TranscriptomeSAM that prevented output to Aligned.toTranscriptome.out.bam of the reads mapped to the very last annotated transcript.

File: resolwe_bio/docker_images/rnaseq/Dockerfile [hunk 2]
```diff
@@ -5,46 +5,31 @@ LABEL MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/reso
...
-    FastQC_VERSION=0.11.5 && \
-    FastQC_SHA256SUM=dd7a5ad80ceed2588cf6d6ffe35e0f161c0d9977ed08355f5e4d9473282cbd66 && \
-    wget -q https://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v$FastQC_VERSION.zip -O /tmp/FastQC.zip && \
-    echo "$FastQC_SHA256SUM */tmp/FastQC.zip" | sha256sum -c - && \
-    mkdir -p /opt/babraham/FastQC-$FastQC_VERSION && \
-    unzip -q /tmp/FastQC.zip -d /opt/babraham/FastQC-$FastQC_VERSION && \
-    rm /tmp/FastQC.zip && \
-    chmod +x /opt/babraham/FastQC-$FastQC_VERSION/FastQC/fastqc && \
-    echo "PATH=\$PATH:/opt/babraham/FastQC-$FastQC_VERSION/FastQC" >> /etc/profile.d/rnaseq.sh && \
+    echo "Installing HISAT2..." && \
```
Comment (tjanez) [inline resolwe_bio/docker_images/rnaseq/Dockerfile]: Break the addition of HISAT2 tool into a separate commit.

**Comment by tjanez** [comment]
> @mzganec @tjanez There is an issue with using a system package which would downgrade the installed STAR aligner tool. In the new releases of STAR some issues were fixed that could affect the analysis we are targeting, e.g.:

> Fixed a bug in --quantMode TranscriptomeSAM that prevented output to Aligned.toTranscriptome.out.bam of the reads mapped to the very last annotated transcript.

@jkokosar, isn't STAR aligner still installed manually?

**Comment by jkokosar** [comment]
It is now, we already reverted this change after your initial review.

**Pull request merged**